lib/repo/git_repository.rb

```diff
@@ -110,25 +110,10 @@ def self.create(connect_string)
       cloned_repo = Git.clone(
         'git@localhost:' + repo_name, MarkusConfigurator.markus_config_repository_storage + '/' + repo_name)
 
-      repo = Rugged::Repository.discover(MarkusConfigurator.markus_config_repository_storage + '/' + repo_name)
-
       # Lets make some sample files and the new master branch
       cloned_repo.reset
       cloned_repo.branch('master')
 
-      # Create the README.md on the filesystem
-      file_path_for_readme = File.join(repo.workdir, 'README.md')
-      File.open(file_path_for_readme, 'w+') do |readme|
-        readme.write('Initial commit.')
-      end
-
-      cloned_repo.add(all: true)
```

Comment:
As an admin, go to the Submissions view for A1.  Click on a link in the Repository column to go to the repository view.  This is where I get the error.

Yes, the changed_filenames_at_path needs to be implemented for git.  (It is something David added for svn to reduce the number of unnecessary revision entries in the drop down menu for selecting a revision.)